src/main/webapp/i18n/de/notification.json

```diff
@@ -10,6 +10,7 @@
             "loadedNotificationCount": "{{loaded}} von {{total}} Mitteilungen geladen",
             "by": "von",
             "allLoaded": "Keine weiteren Mitteilungen vorhanden.",
+            "liveExamExerciseUpdateNotification": "Eine Klausuraufgabe wurde aktualisiert!",
```

Comment:
Because the team prefers active style instead of a passive one.
-> Maybe "Aktualisierte Klausuraufgabe!" would be better?